metaflow/event_logger.py

```diff
@@ -24,6 +24,31 @@ def log(self, payload):
             msg = Message(MessageTypes.BEST_EFFORT, payload)
             self._sidecar.send(msg)
 
+    def log_event(self, msg=None, event_name=None, log_stream=None, other_context=None):
```

Comment:
do we need this method? can we not reuse `def log(self, payload)` where the payload contains the msg, event_name, log_stream and other_context?